hydrosdk/model.py

```diff
@@ -84,9 +84,18 @@ def read_yaml(path):
     return model
 
 
-# TODO: to be implemented
-def read_py(path):
-    pass
+def parse_model_from_json_dict(cluster, json_dict):
+    """
+    Parses json dict which could contain ExternalModel or Model
+    :param cluster:
+    :param json_dict:
+    :return:
+    """
+    is_external = json_dict.get("isExternal", True)
```

Comment:
```
json_dict = {1: 1}
is_external = json_dict.get("isExternal", True)
if is_external:
  print(1)
else:
  print(2)
```
Prints 1

Reply:
That's not a problem, per se. Backend has inconsisted output, depending on the endpoint. With the latest fixes, I tried to make them similar, but there will be situations, when model object is returned without `isExternal` flag. Thus we need some kind of default value.

`True` was a bad option here, I think `False` will be better.

Reply:
@KineticCookie ah, ok. I thought that was unexpected behavior and therefore notified you. In tests some models (LocalModel) were uploaded as external ones though they were uploaded as usual and I thought they were supposed to be class Model in the result.